test/test-v8debugapi.js

```diff
@@ -43,6 +45,67 @@ function stateIsClean(api) {
   return true;
 }
 
+function validateVariable(variable) {
```

Comment:
I think it would need slightly more scrutiny as would want to reason about the backoff strategy. Complete the breakpoint with error? Disable the agent?